ceph-releases/kraken/ubuntu/14.04/daemon/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# DOCKER-VERSION 1.0.0
```

Comment:
I'll like to drop the support of 14.04 and make 16.04 the default for Kraken. Can you remove it and stick with 16.04 as a reference for all the images?